.github/workflows/release.yml

```diff
@@ -0,0 +1,27 @@
+name: Release
+
+on:
+  release:
+    types: published
```

Comment:
Did you actually check if that flag is the one we want?
In my pet-project I am using `released` for this, https://github.com/amureki/statuscheck/blob/master/.github/workflows/release.yml but I can imagine both would work here.

Reply:
hm... @amureki 

I found [this source](https://github.community/t/release-prerelease-action-triggers/17275) for an explanation. 

The differentiator is _pre-release_ vs _release_. 
- `published` is called for both release types 
- `released` is only called for the final release. 

I'll change it